cypress.json

```diff
@@ -3,5 +3,6 @@
   "viewportWidth": 1920,
   "viewportHeight": 1080,
   "defaultCommandTimeout": 10000,
-  "retries": 1
+  "retries": 1,
+  "video": false
```

Comment:
hopefully that speeds things up a bit :)

Reply:
seems like it did!